### ISSUE TITLE
Problem about NAT connect with real machine

### ISSUE THREAD

**Issue by Zyangzzz** [mininet/mininet#1129]
I try to use a mininet connected with a nat node. The nic of the host is 1000Mbps. When I set the mininet's delay of link '0ms'ang the loss rate 0%, the bandwidth between mininet host 'h1' and my another real host is nearly 900Mbps. But when I set the delay '5ms', the bandwidth decreases dramatically.
In fact, no matter how much delay I set, the bandwidth decreases abnormally. I wonder if it has anything to do with OVS.
Finally, I try the same thing above. This time the server is my vm, and the client is the virtual host of mininet running  on the same vm. The result is same as what I did before. 
I wonder if you've had similar problems.
### The python Script of my mininet:
```
from mininet.topo import Topo
from mininet.net import Mininet
from mininet.node import CPULimitedHost
from mininet.link import TCLink
from mininet.util import dumpNodeConnections
from mininet.log import setLogLevel
from mininet.cli import CLI
 
class SingleSwitchTopo(Topo):
    "Single switch connected to n hosts."
    def __init__(self, n=2, **opts):
        Topo.__init__(self, **opts)
        switch = self.addSwitch('s1')
        for h in range(n):
            host = self.addHost('h%s' % (h + 1))
            self.addLink(host, switch,  delay='0ms', loss=0, use_htb=True)
 
def perfTest():
    "Create network and run simple performance test"
    topo = SingleSwitchTopo(n=4)
    net = Mininet(topo=topo,host=CPULimitedHost, link=TCLink)
    net.addNAT().configDefault()
    net.start()
    print ("Dumping host connections")
    dumpNodeConnections(net.hosts)
    print ("Testing network connectivity")
    net.pingAll()
    print ("Testing bandwidth between h1 and h4")
    h1, h4 = net.get('h1', 'h4')
    net.iperf((h1, h4))
    CLI(net)
    net.stop()
 
if __name__=='__main__':
    setLogLevel('info')
    perfTest()
```

### experiment reault (another host - - virtual host h1 ) :
delay = 0ms, loss = 0
![1650290859(1)](https://user-images.githubusercontent.com/43959730/163820088-b5efddd8-b65c-4fc7-8168-4c06f96ba1e3.png)

******
delay = 5ms, loss = 0
![1650290974(1)](https://user-images.githubusercontent.com/43959730/163820200-b2baafe0-682c-4297-96ed-69845d35be68.png)

### experiment reault (vm - - virtual host h1 ) :
delay = 0ms, loss = 0
![1650291028(1)](https://user-images.githubusercontent.com/43959730/163820302-235d6c7c-076e-4050-9e1e-eb8dc0677fd0.png)

******
delay = 5ms, loss = 0
![1650291063(1)](https://user-images.githubusercontent.com/43959730/163820361-4117a105-05a4-418b-a258-026cbd74ec73.png)

I'd be grateful if you could enlighten me!
